### DISCUSSION TITLE
We need your feedback for upcoming SWA improvements!

### DISCUSSION THREAD

**Discussion by thomasgauvin** [Azure/static-web-apps#1439] [General]
Hi all,

We're actively investigating various improvements for the Azure Static Web Apps service and we need your feedback to help prioritize these investments. 

We want to learn more about the workloads you're deploying to Static Web Apps, and how our upcoming improvements would improve your workflow. Specifically, we're looking into increased quotas, facilitating the integration of Static Web Apps within your virtual networks, supporting more regions, enabling data residency use cases, etc. 

Please consider taking 5-10 minutes to fill out this survey to help us shape these future improvements for Azure Static Web Apps: https://microsoft.qualtrics.com/jfe/form/SV_6PbhexfucnNKS7c

Thanks!

The Azure Static Web Apps team

**Comment by kev1010** [comment]
Why is this an issue? Should be a discussion.
